pkg/utils/resources/resources_test.go

```diff
@@ -0,0 +1,386 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
```

Comment:
I don't think that you need to create two separate files for the testing. I think you can just have all of these test go into a file called `suite_test.go` since that's the Ginkgo convention